subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/EclipsePlugin.java

```diff
@@ -275,6 +275,13 @@ public void execute(GenerateEclipseClasspath task) {
                 });
             }
         });
+
+        project.getPlugins().withType(JavaTestFixturesPlugin.class, new Action<JavaTestFixturesPlugin>() {
+            @Override
+            public void execute(JavaTestFixturesPlugin javaTestFixturesPlugin) {
+                model.getClasspath().setContainsTestFixtures(true);
```

Comment:
When this is turned into a property, this should use `convention`.

Reply:
Fixed.